tests/io/test_path.py

```diff
@@ -227,16 +271,13 @@ def test_move_remote(self):
         _to.unlink()
 
     def test_copy_remote_remote(self):
-        # foo = xxx added to prevent same fs token
-        attach("ffs", fs=FakeRemoteFileSystem(auto_mkdir=True, foo="bar"))
-        attach("ffs2", fs=FakeRemoteFileSystem(auto_mkdir=True, foo="baz"))
+        attach("ffs", fs=FakeRemoteFileSystem(skip_instance_cache=True))
+        attach("ffs2", fs=FakeRemoteFileSystem(skip_instance_cache=True))
 
-        dir_src = f"/tmp/{str(uuid.uuid4())}"
-        dir_dst = f"/tmp/{str(uuid.uuid4())}"
+        dir_src = f"bucket1/{str(uuid.uuid4())}"
```

Comment:
I think it would be good to test that we have a `path.bucket` here.

Reply:
Added the asserts